### ISSUE TITLE
[Feature Request] Require conversation resolution

### ISSUE THREAD

**Issue by testworksau** [ossf/allstar#311]
It would be good if this tool could support "Require conversation resolution before merging".

https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/about-protected-branches#require-conversation-resolution-before-merging

We are happy to contribute if you're happy for this setting to be added.

**Comment by jeffmendoza** [comment]
Yes, sounds great. It would be good to have full coverage of all the branch protection options.

**Comment by patflynn** [comment]
+1. Would love to see this feature added. @testworksau were you still planning on picking this up?

**Comment by testworksau** [comment]
No, sorry @patflynn - we are looking at switching to using a combination of the new [GitHub Repository Rules](https://github.blog/changelog/2023-04-17-introducing-repository-rules-public-beta/) for the mandatory rulesets we wish to apply on our organisation(s), and our own tooling that allows status checks to be config-as-code, instead.
